xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -899,7 +974,21 @@
       <Docs>
         <summary>Gets a value that indicates whether access to the collection is synchronized (thread-safe).</summary>
         <value>false in all cases.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.CaptureCollection> instance is cast to an <xref:System.Collections.ICollection> interface.
+ 
+ 
+ 
+ <xref:System.Collections.ICollection.SyncRoot%2A> returns an object, which can be used to synchronize access to the <xref:System.Collections.ICollection>.   
+ Most collection classes in the <xref:System.Collections?displayProperty=nameWithType> namespace also implement a Synchronized method, which provides a synchronized wrapper around the underlying collection.   
+ Enumerating through a collection is intrinsically not a thread-safe procedure. Even when a collection is synchronized, other threads can still modify the collection, which causes the enumerator to throw an exception. To guarantee thread safety during enumeration, you can either lock the collection during the entire enumeration or catch the exceptions resulting from changes made by other threads.   
+
```

Comment:
I think this explanation can be improved, because as it is right now, it seems to apply specifically for the interface method, not for this implemented method.